pkgs/by-name/fl/flow-state/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "flow_state";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "Stan-breaks";
+    repo = pname;
+    rev = "v1.0.3";
+    hash = "sha256-7j8W370lr/QaLL+T7N/2SlcrPe+dTW5zlNPL7+U/Vog=";
+  };
+
+  cargoHash = "sha256-IY4Kd43zLIGRjQbkeZddl6ayRv997HuSKV1DKI8Z6BY=";
+
+  meta = with lib; {
+    description = "A terminal-based habit tracker designed for neurodivergent users";
+    mainProgram = "flow_state";
+    homepage = "https://github.com/Stan-breaks/flow_state";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
From https://github.com/NixOS/nixpkgs/issues/304000
`gpl3` is deprecated in favor of `gpl3Only` and `gpl3Plus`

I'm not a lawyer nor an expert, but I think it's `gpl3Plus` and not `gpl3Only`, since the license contains: "`either version 3 of the License, or
    (at your option) any later version`"